openapi.yaml

```diff
@@ -280,4 +294,116 @@ components:
         type: string
         example: Team A
       example: ["All teams", "Team A", "Team B"]
-
+    EnvironmentsGetResponse:
+      type: object
+      properties:
+        id:
+          title: id
+          description: Environment identifier
+          type: string
+          example: "1"
+        name:
+          title: Name
+          description: Environment name
+          type: string
+          example: DEV
```

Comment:
Should this be  `"DEV"`, because it's a string?

Reply:
I believe that is up to personal preference. YAML is smart enough to distinguish different types and this `DEV` is considered as a string even without the quotes. [Here](https://stackoverflow.com/questions/19109912/yaml-do-i-need-quotes-for-strings-in-yaml) is a StackOverflow explaining the need for quotes. Personally I would leave the quotes out if possible and only use quotes when we have a stringified number (e.g. `"10"`).

Reply:
Thanks for the explanation 💯  
I was thrown off by the examples with `"test-"` but that's because of the special character as I learned now 💡